scipy/stats/_continuous_distns.py

```diff
@@ -3965,6 +4132,11 @@ class geninvgauss_gen(rv_continuous):
     def _argcheck(self, p, b):
         return (p == p) & (b > 0)
```

Comment:
<img width="235" alt="image" src="https://user-images.githubusercontent.com/6570539/158075733-b6ed15d6-b71d-4c1a-95cf-508165d264cd.png">
Probably could be clarified that only b > 0.